webcompat/static/js/lib/bugform.js

```diff
@@ -227,6 +231,18 @@ function BugForm() {
     }
   };
 
+  this.clearStepsToReproducePlaceholder = function() {
+    var stepsToReproduceStrAr = this.stepsToReproduceField
+      .val()
+      .split("4. Nothing happened.\n");
+    // handle the case where stylo pref prepopulated from FF Nightly
+    if (stepsToReproduceStrAr.length > 1) {
```

Comment:
@miketaylr I guess these are 3 cases that I can come up with:
* default case (only the placeholder text)
* servo case (placeholder text + appended servo/stylo pref str)
* updated case (user already clicked and edited the field; ie placeholder text is gone)

I'll rethink how to handle each case